bundles/org.openhab.binding.matter/src/main/java/org/openhab/binding/matter/internal/discovery/MatterDiscoveryHandler.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.matter.internal.discovery;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link MatterDiscoveryHandler}
```

Comment:
Please add a short description

Reply:
I see many classes without a description.
I don't know if you plan to add a description on each class or if we should just forget ?

Reply:
I went through and added descriptions